### ISSUE TITLE
[eyes-storybook] version 3.27.1  throws Error: Cannot find module 'assert/strict'

### ISSUE THREAD

**Issue by EgoziE** [applitools/eyes.sdk.javascript1#728]
This is my error stack:

```
0:55:27]	[npm run test] Error: Cannot find module 'assert/strict'
[20:55:27]	[npm run test] Require stack:
[20:55:27]	[npm run test] - /home/builduser/agent00/work/275f0888d1bce5b0/packages/wix-ui-core/node_modules/@applitools/driver/dist/debug/check-spec-driver.js
[20:55:27]	[npm run test] - /home/builduser/agent00/work/275f0888d1bce5b0/packages/wix-ui-core/node_modules/@applitools/driver/dist/debug/index.js
[20:55:27]	[npm run test] - /home/builduser/agent00/work/275f0888d1bce5b0/packages/wix-ui-core/node_modules/@applitools/eyes-sdk-core/index.js
[20:55:27]	[npm run test] - /home/builduser/agent00/work/275f0888d1bce5b0/packages/wix-ui-core/node_modules/@applitools/eyes-storybook/src/renderStory.js
[20:55:27]	[npm run test] - /home/builduser/agent00/work/275f0888d1bce5b0/packages/wix-ui-core/node_modules/@applitools/eyes-storybook/src/eyesStorybook.js
[20:55:27]	[npm run test] - /home/builduser/agent00/work/275f0888d1bce5b0/packages/wix-ui-core/node_modules/@applitools/eyes-storybook/src/cli.js
[20:55:27]	[npm run test] - /home/builduser/agent00/work/275f0888d1bce5b0/packages/wix-ui-core/node_modules/@applitools/eyes-storybook/bin/eyes-storybook.js
[20:55:27]	[npm run test]     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
[20:55:27]	[npm run test]     at Function.Module._load (internal/modules/cjs/loader.js:725:27)
[20:55:27]	[npm run test]     at Module.require (internal/modules/cjs/loader.js:952:19)
[20:55:27]	[npm run test]     at require (internal/modules/cjs/helpers.js:88:18)
[20:55:27]	[npm run test]     at Object.<anonymous> (/home/builduser/agent00/work/275f0888d1bce5b0/packages/wix-ui-core/node_modules/@applitools/driver/dist/debug/check-spec-driver.js:7:34)
[20:55:27]	[npm run test]     at Module._compile (internal/modules/cjs/loader.js:1063:30)
[20:55:27]	[npm run test]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
[20:55:27]	[npm run test]     at Module.load (internal/modules/cjs/loader.js:928:32)
[20:55:27]	[npm run test]     at Function.Module._load (internal/modules/cjs/loader.js:769:14)
[20:55:27]	[npm run test]     at Module.require (internal/modules/cjs/loader.js:952:19) {
[20:55:27]	[npm run test]   code: 'MODULE_NOT_FOUND',
```

When I roll back to 3.27.0 it works well.
I have a gut feeling it's related to the Storybook v6.4 commit somehow

**Comment by EgoziE** [comment]
It might be this:
https://github.com/applitools/eyes.sdk.javascript1/commit/c4d7dadab2207b7d20825fc0c378805cf19926b1

**Comment by EgoziE** [comment]
Fixed in 3.27.2
